package.json

```diff
@@ -73,6 +73,7 @@
   },
   "license": "MIT",
   "dependencies": {
+    "@types/dateformat": "^5.0.2",
```

Comment:
For some reason version 4 was skipped for this, but I've checked. Version 5 did not change the format masks.